configure.ac

```diff
@@ -3199,10 +3199,14 @@ AC_ARG_ENABLE(loadable-sqlite-extensions,
               AS_HELP_STRING([--enable-loadable-sqlite-extensions],
                              [support loadable extensions in _sqlite module, see Doc/library/sqlite3.rst (default is no)]),
               [],
-              [enable_loadable_sqlite_extensions="no"])
-
+              [enable_loadable_sqlite_extensions=no])
 AC_MSG_RESULT($enable_loadable_sqlite_extensions)
 
+AS_VAR_IF([enable_loadable_sqlite_extensions], [yes], [
+  AC_DEFINE(PY_SQLITE_ENABLE_LOADABLE_EXTENSION, 1,
```

Comment:
Nitpick :) I would prefer `PY_SQLITE_ENABLE_LOADABLE_EXTENSIONS` (with the plural S) or `PY_SQLITE_ENABLE_LOAD_EXTENSION`. Perhaps the latter, as it is shorter.